src/plugins/controls/public/options_list/embeddable/options_list_embeddable.tsx

```diff
@@ -277,73 +278,85 @@ export class OptionsListEmbeddable extends Embeddable<OptionsListEmbeddableInput
+        );
+      if (
+        !selectedOptions ||
+        isEmpty(invalidSelections) ||
+        ignoreParentSettings?.ignoreValidations
+      ) {
+        dispatch(
+          updateQueryResults({
+            availableOptions: suggestions,
+            invalidSelections: undefined,
+            validSelections: selectedOptions,
+            totalCardinality,
+          })
+        );
+      } else {
+        const valid: string[] = [];
+        const invalid: string[] = [];
+
+        for (const selectedOption of selectedOptions) {
+          if (invalidSelections?.includes(selectedOption)) invalid.push(selectedOption);
+          else valid.push(selectedOption);
+        }
+        dispatch(
+          updateQueryResults({
+            availableOptions: suggestions,
+            invalidSelections: invalid,
+            validSelections: valid,
+            totalCardinality,
+          })
+        );
+      }
 
-    // need to get filters, query, ignoreParentSettings, and timeRange from input for inheritance
-    const {
-      ignoreParentSettings,
-      filters,
-      query,
-      timeRange: globalTimeRange,
-      timeslice,
-    } = this.getInput();
-
-    if (this.abortController) this.abortController.abort();
-    this.abortController = new AbortController();
-    const timeRange =
-      timeslice !== undefined
-        ? {
-            from: new Date(timeslice[0]).toISOString(),
-            to: new Date(timeslice[1]).toISOString(),
-            mode: 'absolute' as 'absolute',
-          }
-        : globalTimeRange;
-    const { suggestions, invalidSelections, totalCardinality } =
-      await this.optionsListService.runOptionsListRequest(
-        {
-          field,
-          query,
-          filters,
-          dataView,
-          timeRange,
-          searchString,
-          runPastTimeout,
-          selectedOptions,
-        },
-        this.abortController.signal
-      );
-    if (!selectedOptions || isEmpty(invalidSelections) || ignoreParentSettings?.ignoreValidations) {
-      dispatch(
-        updateQueryResults({
-          availableOptions: suggestions,
-          invalidSelections: undefined,
-          validSelections: selectedOptions,
-          totalCardinality,
-        })
-      );
+      // publish filter
+      const newFilters = await this.buildFilter();
+      batch(() => {
+        dispatch(setLoading(false));
+        dispatch(publishFilters(newFilters));
+      });
     } else {
-      const valid: string[] = [];
-      const invalid: string[] = [];
-
-      for (const selectedOption of selectedOptions) {
-        if (invalidSelections?.includes(selectedOption)) invalid.push(selectedOption);
-        else valid.push(selectedOption);
-      }
       dispatch(
         updateQueryResults({
-          availableOptions: suggestions,
-          invalidSelections: invalid,
-          validSelections: valid,
-          totalCardinality,
+          availableOptions: [],
```

Comment:
Need to set the suggestions back to empty when the search string becomes invalid.